wrap_js/__test__/ConvertMnemonicToSeed.spec.js

```diff
@@ -0,0 +1,54 @@
+import { ConvertMnemonicToSeed } from "../build/Release/cfd_js"
+import TestHelper from "./TestHelper"
+import Bip39DataSource from './data/Bip39DataSource'
+
+const testCase = []
+const testLanguage = ["en"/*, "jp"*/]
+testLanguage.forEach(lang => {
+  const testVectors = Bip39DataSource.GetTestVector(lang)
+  testVectors.forEach((testVector, idx) => {
+    const reqJson = {
+      mnemonic: testVector.mnemonic,
+      passphrase: testVector.passphrase,
+      strictCheck: true,
+      language: lang,
+      useIdeographicSpace: testVector.use_ideographic_space
+    }
+    const expectJson = {
+      seed: testVector.seed,
+      entropy: testVector.entropy
+    }
+
+    testCase.push(TestHelper.createTestCase(
+      `ConvertMnemonicToSeed Test [case ${idx}, lang ${lang}]`,
+      ConvertMnemonicToSeed,
+      [JSON.stringify(reqJson)],
+      JSON.stringify(expectJson)
+    ))
+  })
+})
+
+const errorCase = [
+  TestHelper.createTestCase(
+    "ConvertMnemonicToSeed Error(invalid language)",
+    ConvertMnemonicToSeed,
+    ['{"mnemonic":["abandon","abandon","abandon","abandon","abandon","abandon","abandon","abandon","abandon","abandon","abandon","about"], "passphrase":"TREZOR", "language":"zz"}'],
+    '{"error":{"code":1,"type":"illegal_argument","message":"Failed to ConvertMnemonicToSeed. Not support language passed."}}'
+  ),
+  TestHelper.createTestCase(
+    "ConvertMnemonicToSeed Error(strict check error)",
+    ConvertMnemonicToSeed,
+    ['{"mnemonic":["aa","aa","aa","aa","aa","aa","aa","aa","aa","aa","aa","abort"], "passphrase":"TREZOR", "strict_check": true}'],
+    '{"error":{"code":1,"type":"illegal_argument","message":"Failed to ConvertMnemonicToSeed. Convert mnemonic to entropy error."}}'
+  ),
```

Comment:
● ConvertMnemonicToSeed ErrorCase › ConvertMnemonicToSeed Error(strict check error)

    expect(received).toEqual(expected) // deep equality

    Expected: "{\"error\":{\"code\":1,\"type\":\"illegal_argument\",\"message\":\"Failed to ConvertMnemonicToSeed. Convert mnemonic to entropy error.\"}}"
    Received: "{\"seed\":\"a9bab5924baa4822839f2bded3cbc3bfa138cab7cd678e65552768c4016178d7188c723c26f82ea034484f6f037e4bd98dbf063cb6155802061cd4c1dfaaeb30\"}"